oozie-to-airflow/run-sys-test

```diff
@@ -48,7 +48,7 @@ COMPOSER_LOCATION=$(read_from_file COMPOSER_LOCATION)
 export COMPOSER_LOCATION=${COMPOSER_LOCATION:=europe-west1}
 
 COMPOSER_DAG_BUCKET=$(read_from_file COMPOSER_DAG_BUCKET)
-export COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET:=europe-west1-o2a-integratio-f690ede2-bucket}
+export COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET:=""}
```

Comment:
Yeah. I realized this :). 

I think it will be best if this is combined with the wizard approach where we anyhow should remove all default values anyway (#170),